app/api/schema.py

```diff
@@ -26,6 +30,20 @@ class Query(graphene.AbstractType):
     vendor = graphene.relay.Node.Field(VendorNode)
     vendors = DjangoFilterConnectionField(VendorNode)
 
+    servings = graphene.List(
+        ServingNode,
+        timetable=graphene.String(),
+        vendor=graphene.String(),
+        date=graphene.String()
+    )
+
+    def resolve_servings(self, args, context, info):
+        timetable = Timetable.objects.get(slug=args['timetable'])
+        vendor = Vendor.objects.get(slug=args['vendor'])
+        date = datetime.strptime(args['date'], '%Y-%m-%d').date()
+
+        return ServingAutoUpdate.get_servings(timetable, vendor, date)
+
```

Comment:
I don't think so as `resolve_servings` is meant to override the default behavior of querying the attached model `servings`. And the class property `servings` needs to be there for the same reason others (vendors, users) are there. Let me know if it isn't clear yet. And if probably I didn't get your point, please write out what you are thinking.